smoderp2d/processes/infiltration/__init__.py

```diff
@@ -20,8 +20,8 @@ def set_combinatIndex(newCombinatIndex):
 def philip_infiltration(soil, bil):
     infiltration = combinatIndex[0][3]
     for z in combinatIndex:
-        if ma.all(bil < 0):
-            raise NegativeWaterLevel()
+        # if ma.all(bil < 0):
+        #     raise NegativeWaterLevel()
```

Comment:
What is the purpose of commenting this check? Do we not need to check this anymore?

Reply:
It is done in other place.